### DISCUSSION TITLE
Deleting maskbook account and rejoining later on.

### DISCUSSION THREAD

**Discussion by trymeouteh** [DimensionDev/Maskbook#2121] [General]
If someone decides to delete their maskbook account and unlink their facebook account from maskbook, can they rejoin again using the same maskbook username and using the same facebook account, or will this not be possible?

**Comment by Tedko** [comment]
It’s possible. The user name is not the key here - as long as u r using the
same keypair you can always link back

**Comment by Jack-Works** [comment]
If your private key exists, you can use the same identity to connect to FB or Twitter. If you lost your private key, it's impossible to recover your account. The username you choose actually doesn't matter

**Reply by trymeouteh** [reply]
Is there a maskbook username that is needed to use maskbook?

**Reply by Jack-Works** [reply]
No, the username you input when you set up maskbook is displayed on your device only.

**Reply by trymeouteh** [reply]
> No, the username you input when you set up maskbook is displayed on your device only.

There is no maskbook username saved to a database somewhere? Would this mean the username/account is a file that one would need to backup to be able to restore their maskbook on new devices?

**Reply by yisiliu** [reply]
You can treat the username as a reference in your dashboard, for now. To restore your account, all you need to do is to save the backup file in a safe place. We are going to support email recovery as an option in the future, but this is still not required. You can choose any security level you'd trust.
